docs/source/run/plugin.rst

```diff
@@ -0,0 +1,134 @@
+    ``/var/lib/sopel/plugins``.
+
+If there is no configuration to change for the plugin, you can directly start
+(or restart) your bot and Sopel will load it automatically.
+
+Install packaged plugins
+------------------------
+
+Plugin authors might also publish their works as packages; you can find them by
+searching PyPI, or by using your favorite search engine to search for
+e.g. "sopel reminder plugin".
+
+You can install these packages with ``pip``, the same way it is recommended to
+install Sopel. All you need is to install the plugin for the same Python
+interpreter.
+
+For example, you can install the "remind" plugin from the ``sopel-remind``
+package::
+
+    python3 -m pip install sopel-remind
+
+This will install the plugin from PyPI where you can see `all the releases`__
+of this package.
+
+.. note::
+
+    Each plugin can have specific system dependencies or other requirements
+    that you need to take care of when installing it. Refer to the plugin's
+    documentation for more details.
+
+.. __: https://pypi.org/project/sopel-remind/
+
+
+Enabling or disabling plugins
+=============================
+
+By default, any plugins that Sopel can load will be automatically enabled: it
+is true for all types of plugins (built-in, single file, or packaged plugins).
+You can control this behavior by disabling some plugins, or enabling only some
+plugins.
+
+You can disable a plugin by using the ``sopel-plugins disable`` command line,
+e.g. to disable the remind plugin mentioned before::
+
+    sopel-plugins disable remind
+
+This will add the plugin to the
+:attr:`~sopel.config.core_section.CoreSection.exclude` list. The next time the
+bot starts (or restarts), the plugin won't be loaded.
+
+You can re-enable a plugin by using the ``sopel-plugins enable`` command line,
+e.g. to re-enable the remind plugin::
+
+    sopel-plugins enable remind
+
+Which will remove the plugin from the ``exclude`` list. The plugin will be
+loaded as before the next time the bot starts (or restarts).
+
+Enable-only plugins
+-------------------
+
+By default, all plugins are enabled, and you disable plugins one by one.
+However, you may want to enable only a set of plugins, and disable every other
+one by default.
+
+To do so, the enable command has a ``--allow-only`` option: this will add the
+plugin to the :attr:`~sopel.config.core_section.CoreSection.enable` list::
+
+    sopel-plugins enable --allow-only remind
+
+When you start the bot (or restart), the bot will load the plugins from the
+``enable`` list, and **only these plugins**.
+
+
+Configuring plugins
+===================
+
+A plugin can define a configuration section so you can configure its behavior.
+If the plugin supports it, you may be able to use the configuration wizard to
+edit the configuration of the plugin::
+
+    sopel-plugins configure remind
+
+Alternatively, you can edit your configuration file and follow the plugin's
+documentation to set the appropriate options and values:
```

Comment:
```suggestion
Alternatively, you can edit your configuration file and follow the plugin's
documentation to set the appropriate options and values. Here's an
example configuration of the `sopel-remind plugin`__, which defines the
``[remote]`` section for its own use:

.. __: https://github.com/sopel-irc/sopel-remind

```

Reply:
Nit nit: `sopel-remind` defines the `[remind]` section for its own use, not the `[remote]` section.

Reply:
I'm not sure we should add a link on "sopel-remind". Or maybe a link to the [pypi's release page](https://pypi.org/project/sopel-remind/) and not the github one, as it is a plugin that should be installed through pip.

Reply:
`extlinks` extension can make PyPI links easy, btw. Think <tt>:pypi:\`packagename\`</tt>.

Since the last top-level reply from @Exirel mentions needing to rebase, you'll get the newly enabled `extlinks` extension out of it as a silver lining (both the `configuration.rst` conflict and the new extension landed on `master` from #2550).